### PR TITLE
Halloween Update to names.js

### PR DESCRIPTION
**Pull request by SpoopyCaboose**
This is the Halloween version of the random name list.

### CODE REVIEW

**Comment by intOrfloat** [comment]
YAP

**Comment by TerriDebbi** [comment]
spoopy
halloween_hades

**Comment by Squarific** [comment]
I'll add this when its closer to halloween. You should not have renamed the file though.

**Comment by SpoopyCaboose** [comment]
Understood. Thanks :)

File: server/realTime/scripts/names_Halloween.js [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+  "Tree",
+  "Destroyer",
+  "Fester",
+  "Forest",
+  "Glenda",
+  "Gloria",
+  "Ingrid",
+  "Spook",
+  "Ursula",
+  "Voilet",
```
Comment (TerriDebbi) [inline server/realTime/scripts/names_Halloween.js]: Violet not Voilet

File: server/realTime/scripts/names_Halloween.js [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+  "Damian",
+  "Dolly",
+  "Dante",
+  "Igor",
+  "Inferno",
+  "Theron",
+  "Jasper",
+  "Storm",
+  "Vlad",
+  "Uber",
```
Comment (TerriDebbi) [inline server/realTime/scripts/names_Halloween.js]: rekt

File: server/realTime/scripts/names_Halloween.js [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+  "Fawn",
+  "Dior",
+  "Gem",
+  "Jewel",
+  "Noire",
+  "Opal",
+  "Onyx",
+  "Yew",
+  "Ruby",
+  "Saphire",
```
Comment (TerriDebbi) [inline server/realTime/scripts/names_Halloween.js]: Sapphire not Saphire

File: server/realTime/scripts/names_Halloween.js [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+  "Ebony",
+  "Skeleton",
+  "Garnet",
+  "Alpha",
+  "Omega",
+  "Mauve",
+  "Twinkle",
+  "Star",
+  "Dante",
+  "Crimson",
```
Comment (TerriDebbi) [inline server/realTime/scripts/names_Halloween.js]: Crismon not Crimson

File: server/realTime/scripts/names_Halloween.js [hunk 5]
```diff
@@ -0,0 +1,305 @@
...
+  "Dior",
+  "Gem",
+  "Jewel",
+  "Noire",
+  "Opal",
+  "Onyx",
+  "Yew",
+  "Ruby",
+  "Saphire",
+  "Amythest",
```
Comment (TerriDebbi) [inline server/realTime/scripts/names_Halloween.js]: Amethyst*

File: server/realTime/scripts/names_Halloween.js [hunk 6]
```diff
@@ -0,0 +1,305 @@
...
+  "Thief",
+  "Dragon",
+  "Shadow",
+  "Crow",
+  "Jack",
+  "Halloweena",
+  "Wizard",
+  "Knight",
+  "Fairy",
+  "Princess",
```
Comment (TerriDebbi) [inline server/realTime/scripts/names_Halloween.js]: 236 is the same

File: server/realTime/scripts/names_Halloween.js (hunk 5 above)
Comment (SpoopyCaboose) [inline server/realTime/scripts/names_Halloween.js]: Are you having fun?

**Comment by Squarific** [comment]
Added via https://github.com/Squarific/Anondraw/pull/110

**Pull request closed without merging**